Hazel/src/Hazel/Core.h

```diff
@@ -18,6 +18,9 @@
 
 #ifdef HZ_DEBUG
 	#define HZ_ENABLE_ASSERTS
+	#define HZ_ENABLE_RENDERER_LOG
```

Comment:
Why is this PR doing multiple things?

Reply:
So you would always log the openGL debug callback?

Reply:
It might indeed look beter to seperate this into two seperate PRs... so then keep this one as the OpenGL debug callback, and another one maybe for these macro definitions (after this one is merged)